airbyte-integrations/connectors/destination-bigquery/src/main/java/io/airbyte/integrations/destination/bigquery/BigQueryStagingConsumerFactory.java

```diff
@@ -88,7 +88,7 @@ private CheckedConsumer<AirbyteStreamNameNamespacePair, Exception> incrementalTy
     return (streamId) -> {
       if (!valve.containsKey(streamId)) {
         valve.addStream(streamId);
-      } else if (valve.readyToTypeAndDedupe(streamId)) {
+      } if (valve.readyToTypeAndDedupe(streamId)) {
```

Comment:
Note to @edgao - maybe a similar thing needs to change for Snowflake

Reply:
ah is this because we have to type+dedupe the first time we see each stream?

also style nit, put the `if` on a new line: (i'm surprised the linter didn't catch this)
```java
} 
if (valve.readyToTypeAndDedupe(streamId)) {
```